hooks/generatePdf.js

```diff
@@ -0,0 +1,30 @@
+const puppeteer = require('puppeteer');
+const path = require('path');
+
+module.exports = {
+    'generate:after': generatePdf
+};
+
+async function generatePdf(generator) {
+    const targetDir = generator.targetDir;
+    const parameters = generator.templateParams;
+    //all actions of this hook depend on parameters passed by the user, if non are provided we should just stop the hook
+    if (!parameters) return;
```

Comment:
refactor this `if` and remove `if` from line 27.
in my template I had it done this way as the hook supported 3 different parameters. In case of this hook, it is just this parameter. So basically here in line 12 you should return not only if there are no parameters but also if `parameters.pdf` is not provided

Reply:
Oh yeah that makes sense, will do the changes